examples/rn-connection-and-error/frontend/app/screens/StoreScreen.tsx

```diff
@@ -72,6 +84,11 @@ export function StoreScreen() {
             />
           </View>
           <View style={styles.triggers}>
+            <View style={styles.status}>
+              <Text style={styles.statusText}>
+                Team: {user.customData.team}
+              </Text>
+            </View>
```

Comment:
Nit: Just moved this above the "Connected" text.

But also, removed the `?` after `customData` as it will always return a record.